doc/source/client.rst

```diff
@@ -191,14 +191,9 @@ The logical devices represented by the device is addressed with the :mod:`slave=
 With **Serial**, the comm port is defined when creating the object.
 The physical devices are addressed with the :mod:`slave=` parameter.
 
-:mod:`slave=0` is used as broadcast in order to address all devices.
-However experience shows that modern devices do not allow broadcast, mostly because it is
-inheriently dangerous. With :mod:`slave=0` the application can get upto 254 responses on a single request,
-and this is not handled with the normal API calls!
-
-The simple request calls (mixin) do NOT support broadcast, if an application wants to use broadcast
-it must call :mod:`client.execute` and deal with the responses.
+:mod:`slave=0` is used as broadcast in order to address all devices. To accommodate non-standard behaviour of devices, all request calls to :mod:`slave=0` will expect and wait for at least one response. If multiple devices on the bus are expected to respond to broadcast requests (for example for the purpose of device detection), the application can get upto 254 responses on a single request, and this is not handled with the normal API calls. If an application is expecting multiple responses to a broadcast request, it must call :mod:`client.execute` and deal with the responses.
```

Comment:
used as broadcast (in accordance with the modbus standard)

Not "at least" it is "wait for one response".